### DISCUSSION TITLE
Can I use same logger for agent and application?

### DISCUSSION THREAD

**Discussion by zmes50416** [open-telemetry/opentelemetry-java-instrumentation#7257] [Q&A]
When I run my application with the agent, I found two different style of logs in the console.
I notice that agent itself is using a SLF4J's SimpleLogger for it internal logging, but my application is using logback & JSON encoder to create log. Since I need to index the log by JSON fields to some internal search engine, these agent's internal log will not be indexed because of the format different. 

So is it possible to allows agent to use the same logging framework or share the same logging format?

**Comment by laurit** [comment]
No, it is not currently possible.

**Reply by zmes50416** [reply]
I see, still thank you for quick reply.
